go-controller/pkg/node/gateway_localnet.go

```diff
@@ -76,6 +76,11 @@ func newLocalGateway(nodeName string, hostSubnets []*net.IPNet, gwNextHops []net
 			if err != nil {
 				return err
 			}
+			if config.Gateway.DisableForwardingOnInterfaces {
```

Comment:
when `DisableForwardingOnInterfaces` is false don't we still want to add the drop rule you initially proposed?
`-A FORWARD -i breth0 -o breth0 -j DROP`
Otherwise the issue with looping traffic will still be there when `DisableForwardingOnInterfaces` is not set

Reply:
So upstream we don't plan to fix that bug. We are going to fix it downstream where DisableForwardInterfaces is going to be true.